src/common/POLFields/FieldsLocation/validate.js

```diff
@@ -42,6 +45,10 @@ export const validateNotNegative = value => {
 };
 
 export const validateQuantityPhysical = (value, { cost, locations = [], physical, orderFormat }) => {
+  const validateNotNegativeMessage = validateNotNegative(value);
```

Comment:
since validation for fields can't be array in react-final-form we have to compose required and specific validators manually.